nix/vend/remove_tidy.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cli/cmd.go b/cli/cmd.go
+index c766559..3a133fd 100644
+--- a/cli/cmd.go
++++ b/cli/cmd.go
+@@ -12,7 +12,7 @@ import (
+ // UpdateModule makes sure the module is updated ready to vendor the
+ // dependencies.
+ func UpdateModule() {
+-	var commands = []string{"tidy", "download", "vendor"}
++	var commands = []string{"download", "vendor"}
```

Comment:
Out of curiosity, why is this patch necessary?

Reply:
I think only @balsoft can answer this question

Reply:
It's needed because the results of `tidy` are not generally reproducible (IIRC it depends on latest versions of some packages). So we apply this patch in order to make `vend` reproducible.